server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/ScheduleController.java

```diff
@@ -91,6 +96,12 @@ public ListResponse<String> getDownloadUrl(@PathVariable Long id, @RequestBody L
         return Responses.list(scheduleService.getAsyncDownloadUrl(id, objectId));
     }
 
+    @RequestMapping(value = "/{id:[\\d]+}/jobs/dlm/limiterConfig", method = RequestMethod.PUT)
+    public SuccessResponse<DlmLimiterConfig> updateLimiterConfig(@PathVariable Long id,
```

Comment:
no sure the scope of rate limiting, it is schedule level or job level ?

if schedule level, url may `/schedules/{scheduleIdd}/dlmRateLimitConfiguration`